src/webhooks/types/ServiceList.ts

```diff
@@ -0,0 +1,44 @@
+/* tslint:disable */
```

Comment:
Please `rm -rf` whole `src/webhooks/types` and build fresh and clean types. This file shouldn't even be here.